neo/io/nestio.py

```diff
@@ -482,8 +500,8 @@ def read_segment(self, gid_list=None, time_unit=pq.ms, t_start=None,
         gid_list : list, default: None
             A list of GDF IDs of which to return SpikeTrain(s). gid_list must
             be specified if the GDF file contains neuron IDs, the default None
-            then raises an error. Specify an empty list [] to retrieve the spike
-            trains of all neurons.
+            then raises an error. Specify an empty list [] to retrieve the
```

Comment:
This help text is not complete: The code also accepts tuples of the form (a,b), which are expanded to `range(a,b)`. I am not sure this is particularly helpful, I would suggest to drop this code below (line 538), rather than adapt this help text to include this case.